packaging/linux/deb/openra.lintian-overrides

```diff
@@ -0,0 +1,3 @@
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSans.ttf also in fonts-freefont-ttf
+openra: duplicate-font-file usr/lib/openra/mods/common/FreeSansBold.ttf also in fonts-freefont-ttf
+openra: binary-without-manpage usr/games/openra-server
```

Comment:
Okay, fair enough. I think we don't see any Lintian output as our CI scripts don't run it. Thanks for making us aware this way.